site/frontend/src/utils/copy.ts

```diff
@@ -0,0 +1,3 @@
+export function deepCopy<T>(value: T): T {
+  return JSON.parse(JSON.stringify(value));
```

Comment:
We would have to be careful about this if the filter ever stores Dates.

Reply:
Good point. I didn't want to include another dependency just for this one function. I suppose that this filter shouldn't care about dates, there's another filter for that.